test/unit/auth/Logout.test.ts

```diff
@@ -0,0 +1,31 @@
+import { Logout } from '../../../src/auth/domain/useCases/Logout';
+import { assert, createSandbox, SinonSandbox } from 'sinon';
+import { IAuthRepository } from '../../../src/auth/domain/repositories/IAuthRepository';
+import { WriteError } from '../../../src/core/domain/repositories/WriteError';
+
+describe('execute', () => {
+  const sandbox: SinonSandbox = createSandbox();
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  test('should not return error on repository success', async () => {
+    const authRepositoryMock = <IAuthRepository>{};
+    authRepositoryMock.logout = sandbox.mock();
+    const sut = new Logout(authRepositoryMock);
+    await sut.execute();
+  });
+
+  test('should return error result on repository error', async () => {
+    const authRepositoryStub = <IAuthRepository>{};
+    const testWriteError = new WriteError();
+    authRepositoryStub.logout = sandbox.stub().throwsException(testWriteError);
+    const sut = new Logout(authRepositoryStub);
+
+    let actualError: WriteError = undefined;
```

Comment:
VS Code is complaining about this line:

![Screen Shot 2023-04-26 at 9 06 10 AM](https://user-images.githubusercontent.com/21006/234584214-09a6c1ae-6693-4fd9-b561-a40435a932c7.png)